src/mococrw/openssl_wrap.h

```diff
@@ -556,7 +556,9 @@ void _X509_REQ_sign_ctx(X509_REQ* req, EVP_MD_CTX* ctx);
  * or hash computations.
  */
 enum class DigestTypes {
+    SHA1,
     SHA256, // @MARCUS: Shouldn't there be some value initialization with openssl consts here?
```

Comment:
If remember correctly this @MARCUS comment is to be removed.

Reply:
That comment was raised in another code review. Comment will be removed if requested in external review.